### PR TITLE
wdio-lambdatest-service: Updated docs for New service Lambdatest #5057

### PR DESCRIPTION
**Pull request by kanhaiya15**
## Proposed changes
Docs changes for service lambdatest

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ If you're looking for issues to help out with, check out [the issues labelled "g
 - [@wdio/selenium-standalone-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-selenium-standalone-service) - A WebdriverIO service that automatically sets up a selenium standalone server
 - [@wdio/browserstack-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-browserstack-service) - A WebdriverIO service that provides a better integration into Browserstack
 - [@wdio/crossbrowsertesting-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-crossbrowsertesting-service) - A WebdriverIO service that provides a better integration into CrossBrowserTesting
+- [wdio-lambdatest-service](https://github.com/LambdaTest/wdio-lambdatest-service) - A WebdriverIO service that provides a better integration into LambdaTest
```
Comment (christian-bromann) [inline README.md]: Given you host the service in your organisation we should remove this here.
```suggestion
```

File: scripts/docs-generation/3rd-party/services.json [hunk 2]
```diff
@@ -47,4 +47,10 @@
     "githubUrl": "https://github.com/chmanie/wdio-intercept-service",
     "npmUrl": "https://www.npmjs.com/package/wdio-intercept-service"
   }
```
Comment (christian-bromann) [inline scripts/docs-generation/3rd-party/services.json]:
```suggestion
  },
```

File: README.md (hunk 1 above)
Comment (kanhaiya15) [reply README.md]: Removed

File: scripts/docs-generation/3rd-party/services.json (hunk 2 above)
Comment (kanhaiya15) [reply scripts/docs-generation/3rd-party/services.json]: Added

**Comment by kanhaiya15** [comment]
Thanks @christian-bromann

**Pull request merged**